app/_hub/kong-inc/acme/index.md

```diff
@@ -115,7 +115,23 @@ verify Let's Encrypt API. The CA-bundle file is usually `/etc/ssl/certs/ca-certi
 Ubuntu/Debian and `/etc/ssl/certs/ca-bundle.crt` for CentOS/Fedora/RHEL.
 
 #### Enable the Plugin
+
+For all the domains that you need to get certificate, make sure `DOMAIN/.well-known/acme-challenge`
```

Comment:
It sounds like you need to map this for each domain - if that's right, then use this phrasing:

```suggestion
For each the domain that needs a certificate, make sure `DOMAIN/.well-known/acme-challenge`
```